Source/carthage/Archive.swift

```diff
@@ -23,6 +23,9 @@ public struct ArchiveCommand: CommandType {
 
 				return SignalProducer(values: Platform.supportedPlatforms)
 					|> map { platform in platform.relativePath.stringByAppendingPathComponent(options.frameworkName).stringByAppendingPathExtension("framework")! }
+					|> flatMap(.Concat) { relativePath in
+						return SignalProducer(values: [relativePath, relativePath.stringByAppendingPathExtension("dSYM")!])
```

Comment:
Could you combine this with the previous `map` so it's done in one step instead of 2?